src/network/networkd-ndisc.h

```diff
@@ -56,9 +53,7 @@ typedef struct NDiscDNR {
         sd_dns_resolver resolver;
 } NDiscDNR;
 
-static inline char* NDISC_DNSSL_DOMAIN(const NDiscDNSSL *n) {
-        return ((char*) n) + ALIGN(sizeof(NDiscDNSSL));
-}
+char* ndisc_dnssl_domain(const NDiscDNSSL *n);
```

Comment:
Not your fault, but when you touch this, please constify the return value.

Reply:
This is actually depended on to be writable in ndisc_router_process_dnssl()

Reply:
Indeed...